.github/workflows/tests.yml

```diff
@@ -12,6 +12,11 @@ on:
   pull_request:
     branches-ignore:
       - v[0-9]+.[0-9]+.[0-9x]+-doc
+    paths-ignore:
+      # No need to test on documentation changes
```

Comment:
I find the wording here a bit ambiguous. Maybe "skip running tests if changes are only in the documentation directories"?

Reply:
Good suggestion, thanks.